src/DistributedLock/DistributedLock.csproj

```diff
@@ -53,7 +53,7 @@
     <ProjectReference Include="..\DistributedLock.Redis\DistributedLock.Redis.csproj" />
     <ProjectReference Include="..\DistributedLock.ZooKeeper\DistributedLock.ZooKeeper.csproj" />
     <ProjectReference Include="..\DistributedLock.MySql\DistributedLock.MySql.csproj" />
-    <ProjectReference Include="..\DistributedLock.Oracle\DistributedLock.Oracle.csproj" Condition="'$(TargetFramework)' != 'netstandard2.0' AND '$(TargetFramework)' != 'net461'" />
```

Comment:
` TargetFrameworks` is `netstandard2.0;netstandard2.1;net462 in this project and `netstandard2.1;net462` in `DistributedLock.Oracle`. Therefore, we only need to filter out `netstandard2.0`.